src/pytsql/grammar/adjust_antlrs_grammar.py

```diff
@@ -0,0 +1,76 @@
+import click
+from pathlib import Path
+import re
+
+PROTECTED_RULE_ELEMENT_LABELS = [
+    "from",
+    "id",
+    "property",
+    "str",
+    "format",
+    "input",
+    "with",
+    "type",
+    "file"
+]
+CONFLICTING_NAMES_ON_WINDOWS = [
+    "PLATFORM"
+]
+
+# CLI rename_protected_rule_element_labels
+@click.group()
```

Comment:
Just curiosity, why do you create two groups with one command each instead of just one group with the commands